tests/nlu/extractors/test_entity_synonyms.py

```diff
@@ -11,7 +11,7 @@ def test_entity_synonyms():
         {"entity": "test", "value": "china", "start": 0, "end": 6},
     ]
     ent_synonyms = {"chines": "chinese", "NYC": "New York City"}
-    EntitySynonymMapper(synonyms=ent_synonyms).replace_synonyms(entities)
+    EntitySynonymMapperComponent(synonyms=ent_synonyms).replace_synonyms(entities)
```

Comment:
The test after this one seems to use the old component architecture and I'm not quite sure what to do with it - is that an important part of the test or should I strip it out and essentially replicate this?

Reply:
Oh dear - the test is indeed very confusing. It currently gets a pipeline, then extracts the one component which we need and then trains it 😮‍💨 I think you can drop all of this and just create a new `EntitySynonymMapperComponent` and call train on it